API_v2.md

```diff
@@ -2065,45 +2065,36 @@ The balance for an account, as well as some information about syncing status nee
 
 * [get_wallet_status](#get-wallet-status)
 
-### The Assigned Address Object
+### The Address Object
 
 #### Attributes
 
 | *Name* | *Type* | *Description*
 | :--- | :--- | :---
-| address_id | string | Unique identifier for the address.
-| account_id | string | Unique identifier for the assigned associated account.
+| object | string, value is "address" | String representing the object's type. Objects of the same type share the same value.
 | public_address | string | Shareable B58 encoded string that represents this address.
-| address_book_entry_id | serialized id | The id for an Address Book Entry object if associated to the address.
-| comment | string | An arbitrary string attached to the object.
-
-#### More Attributes
-
-| *Name* | *Type* | *Description*
-| :--- | :--- | :---
-| object | string, value is "assigned_address" | String representing the object's type. Objects of the same type share the same value.
+| account_id | string | Unique identifier for the assigned associated account.
+| metadata | string | An arbitrary string attached to the object.
 | subaddress_index | string (uint64) | The assigned subaddress index on the associated account.
 | offset_count | int | The value to offset pagination requests for assigned_address list. Requests will exclude all list items up to and including this object.
```

Comment:
I understand what the offset is doing, but what is the page size?

Reply:
This is still yet to be implemented, but is plumbing for the pagination options